curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64faca774fd9fd74bc084cc9.md

```diff
@@ -16,7 +16,7 @@ To begin, within the `closeTaskFormBtn` event listener, create a `formInputsCont
 You should use `const` to create a variable `formInputsContainValues` with the value `titleInput.value || dateInput.value || descriptionInput.value;`
 
 ```js
-assert.match(code, /const\s+formInputsContainValues\s*=\s*titleInput\.value\s*\|\|\s*dateInput\.value\s*\|\|\s*descriptionInput\.value;?/)
+assert.match(code, /const\s+formInputsContainValues\s*=\s*(titleInput\.value\s*(\|\||!==\s*['"]?null['"]?)\s*|\s*dateInput\.value\s*(\|\||!==\s*['"]?null['"]?)\s*|\s*descriptionInput\.value\s*(\|\||!==\s*['"]?null['"]?)\s*)+;?/)
```

Comment:
It seems to match even just the first conditional:
```javascript
const formInputsContainValues = titleInput.value !== null
```

```suggestion
assert.match(code, /const\s+formInputsContainValues\s*=\s*titleInput\.value\s*(!==\s*null)?\s*\|\|\s*dateInput\.value\s*(!==\s*null)?\s*\|\|\s*descriptionInput\.value\s*(!==\s*null)?\s*;?/)
```

- - -

As a side note - it's not necessary at this point to update the branch with latest changes from `main`. Unless there will appear merge conflicts, the PR will be merged without issues. All it does right now it basically launching CI checks again.